docker/containerbuild.sh

```diff
@@ -15,10 +15,23 @@
 USAGE="USAGE: $(basename "$0") kickstart"
 KICKSTART="$1"
 KSNAME=${KICKSTART%.*}
+KSVER=$(expr "${KSNAME}" : '.*\-\([0-9\.]*\).*')
 BUILDDATE=$(date +%Y%m%d)
 BUILDROOT=/var/tmp/containers/$BUILDDATE/$KSNAME
 CONT_ARCH=$(uname -m)
 
+declare -A BOOTISO_URLS
+BOOTISO_URLS=( \
+    ["5.11x86_64"]="http://mirror.centos.org/centos/5.11/os/x86_64/images/boot.iso" \
+    ["5x86_64"]="http://mirror.centos.org/centos/5/os/x86_64/images/boot.iso" \
+    ["6x86_64"]="http://mirror.centos.org/centos/6/os/x86_64/images/boot.iso" \
+    ["7arm64"]="http://mirror.centos.org/altarch/7/os/aarch64/images/boot.iso" \
+    ["7ppc64le"]="$(curl -L http://mirror.centos.org/altarch/7/isos/ppc64le 2>&1 | sed -n -r -s 's,.*(CentOS.*NetInstall.*\.iso).*,http://mirror.centos.org/altarch/7/isos/ppc64le/\1,gp')" \
+    ["7x86_64"]="http://mirror.centos.org/centos/7/os/x86_64/images/boot.iso" \
+)
```

Comment:
As of CentOS 7.3, the http://mirror.centos.org/altarch/7/os/ppc64le/images/boot.iso is available for ppc64le.

@mkumatag can you please update this pull request with the new boot.iso location?